doc/changelog/05-Ltac-language/16661-lia-conversion.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  When the `Psatz Conversion` flag is set (disabled by default), the tactics :tacn:`lia`, :tacn:`nia`, :tacn:`lra`, :tacn:`nra`
+  and :tacn:`psatz` are using convertibility instead of syntactic equality during syntaxification.
```

Comment:
```suggestion
  A `Psatz Conversion` flag to control whether to use :term:`definitional equality`
  (:term:`convertibility <convertible>`) or syntactic equality during syntaxification
  for the tactics :tacn:`lia`, :tacn:`nia`, :tacn:`lra`, :tacn:`nra` and :tacn:`psatz`.
```

I have no idea what "syntaxification" means.  Can you find better wording?
I think no need to mention the default here.